includes/controls/class-kirki-controls-repeater-control.php

```diff
@@ -495,6 +495,16 @@ public function repeater_js_template() {
 										<# } #>
 									</div>
 
+								<# } else if ( 'custom' === field.type ) { #>
+
+									<# if ( field.label ) { #>
+										<span class="customize-control-title">{{ field.label }}</span>
+									<# } #>
+									<# if ( field.description ) { #>
+										<span class="description customize-control-description">{{ field.description }}</span>
+									<# } #>
+									<div data-field="{{{ field.id }}}">{{ field.default }}</div>
```

Comment:
thanks. It is now working. <div data-field="{{{ field.id }}}">{{{ field.default }}}</div>

I can now output html in the repeater field.